client/dashboard/index.js

```diff
@@ -62,15 +63,21 @@ class Dashboard extends Component {
 		}
 
 		const productIds = this.getProductIds();
+		const backUrl = getAdminLink( getNewPath( {}, '/', {} ) );
+		const { connectNonce } = getSetting( 'onboarding', {} );
+
 		if ( ! productIds.length ) {
 			return;
 		}
 
 		document.body.classList.add( 'woocommerce-admin-is-loading' );
 
 		const url = addQueryArgs( 'https://woocommerce.com/cart', {
-			'wccom-back': window.location.href,
+			'wccom-site': getSetting( 'siteUrl' ),
+			'wccom-woo-version': getSetting( 'wcVersion' ),
 			'wccom-replace-with': productIds.join( ',' ),
+			'wccom-connect-nonce': connectNonce,
+			'wccom-back': backUrl,
```

Comment:
Even though this is what we had before, I seem to be having issues with `backUrl` containing the protocol and site url. The redirect ends up breaking on the WooCommerce.com side. Replacing this with just the `path` seems to fix it.

Reply:
Updated this to just the path.  Can you check and let me know if that fixes it on your end, @justinshreve?

I'm getting an `updateProfileItems()` loop at the end of the theme step on a Jurassic Ninja site the second time going through the profiler, but not sure if that's related to this PR or not.